app/controllers/kaui/accounts_controller.rb

```diff
@@ -54,6 +54,17 @@ def new
   end
 
   def create
+
+    test_phone = KauiPhoneValid.new
+    test_phone.phone = params[:phone]
+
+
+    unless test_phone.valid?
+      flash.now[:error] = "Phone number is invalid"
```

Comment:
Should this be something like `test_phone.errors.messages` to pick up the `global.errors.phone_format` from the model?